docs/notebooks/regional_frequency_analysis.ipynb

```diff
@@ -0,0 +1,970 @@
+{
```

Comment:
<blockquote>For the regional analysis, we need to resample all stations, but, this time, it's much faster as no fit is involved.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>